lib/testLauncher/SuitestLauncher.js

```diff
@@ -127,7 +127,10 @@ class SuitestLauncher {
 		validateInput('INTERACTIVE', this.ownArgv);
 
 		try {
-			const {username, password, orgId, deviceId, appConfigId, inspect, inspectBrk, logDir} = this.ownArgv;
+			const {
+				username, password, orgId, deviceId, appConfigId, logDir,
+				inspect, inspectBrk, debugBrk,
```

Comment:
We do not support debugBrk, it's deprecated

Reply:
Ok, we should change docs description about vs-code dbugging

Reply:
I'll create a ticket in Jira for this